locale/en/get-involved/node-meetups.md

```diff
@@ -386,6 +386,17 @@ REQUIREMENTS
 - Organizer name - Ryan Connelly
 - Organizer contact info -
 
+#### Florida
```

Comment:
For lexicographical order, this should be put above Georgia.

Reply:
Helped to fix

Reply:
Ooops, I mean this whole addition(including the meetup) should be put above `Georgia`.😅

Jacksonville is in Florida

![screen shot 2019-03-01 at 9 03 00 pm](https://user-images.githubusercontent.com/23313266/53639867-6f689880-3c65-11e9-8bbc-dbbb9e907a98.png)

Reply:
OK